js/app.js

```diff
@@ -0,0 +1,91 @@
+// sign-in input validator
+
+const mailInput = document.querySelector(".sign-up__form--input input");
+const mailButton = document.querySelector(".sign-up__form--input button");
+const inputContainer = document.querySelector(".sign-up__form--input");
+const newAlert = document.createElement("h5");
+
+const inputValidator = () => {
+  if (
+    mailInput.value.indexOf("@") !== -1 &&
+    mailInput.value.indexOf(".") !== -1 &&
+    mailInput.value.length > 6
+  ) {
+    mailInput.style.border = "1px solid green";
+  } else {
+    mailInput.style.removeProperty("border");
+  }
+
+  if (mailButton.nextElementSibling === newAlert) {
+    newAlert.parentElement.removeChild(newAlert);
+  }
+};
+
+mailInput.addEventListener("input", inputValidator);
+
+// sign-in button validator
+
+let counter = 0; // counter of clicks
+const buttonValidator = () => {
+  counter++;
+  if (
+    mailInput.value.indexOf("@") === -1 ||
+    mailInput.value.indexOf(".") === -1 ||
+    (mailInput.value.length <= 6 && counter >= 1)
+  ) {
+    mailInput.value = "";
+    mailInput.style.border = "1px solid red";
+    inputContainer.appendChild(newAlert);
+    newAlert.innerText = "Type your email correctly!";
+    newAlert.style.color = "red";
+  } else if (
+    mailInput.value.indexOf("@") !== -1 &&
+    mailInput.value.indexOf(".") !== -1 &&
+    mailInput.value.length > 6
+  ) {
+    mailInput.value = "";
+    alert("Congratulations! You've just signed up!");
+  }
+};
+
+mailButton.addEventListener("click", buttonValidator);
+
+// loading more posts
+
+const loadMoreButton = document.querySelector(
+  ".posts-second__load-more button"
+);
+const loadMoreContainer = document.querySelector(".posts-second__load-more");
+const postsSecondSection = document.querySelector(".posts-second .row");
+
+const loadMore = () => {
```

Comment:
To pewnie jeszcze nie na tym etapie, ale i tak Ci się przyda:

Funkcje powinny mieć pojedynczą odpowiedzialność. Zwróć uwagę ile w tej funkcji dzieje się rzeczy:
- tworzysz całą masę elementów
- modyfikujesz te elementy
- dodajesz je do DOMu

Najlepiej byłoby wydzielić z tej funkcji kilka mniejszych, np.:
1. funkcję tworzącą elementy i wrzucającą je do arrayu
2. funkcję iterującą przez elementy i nadającą im określone style
3. w końcu funkcję dodającą elementy do DOMu

Do tego ta nazwa jest bardzo generyczna. Nazwy mogą być długie, bo ważne jest to, żeby jak najwięcej opisywały